src/components/breadcrumb/breadcrumb.stories.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+import { Breadcrumb } from './breadcrumb';
+import Header from '../header/header';
+import { GlobalStyle } from '../layout/global-style';
+import AngularIMG from '../../../blog-posts/images/angular.png';
+
+export default {
+  component: Breadcrumb,
+  title: 'Components/Breadcrumb',
+  decorators: [
+    (Story) => (
+      <>
+        <GlobalStyle $lightTheme={true} />
+        <Story />
+      </>
+    ),
+  ],
+  parameters: {
+    layout: 'fullscreen',
+  },
+} as ComponentMeta<typeof Breadcrumb>;
+
+const Template: ComponentStory<typeof Breadcrumb> = (args) => (
+  <Breadcrumb {...args} />
+);
+const CustomHeader = (transparent = false) => (
+  <Header
+    siteTitle={'Satellytes'}
+    transparent={transparent}
+    $lightTheme={true}
+  />
+);
+
+export const Regular = Template.bind({});
+Regular.args = {
+  breadcrumbEntries: [
+    { pathname: '/', crumbLabel: 'main' },
+    { pathname: '/blog', crumbLabel: 'blog' },
+  ],
+};
+
+export const WithCustomLabel = Template.bind({});
+WithCustomLabel.args = {
+  breadcrumbEntries: [
+    { pathname: '/', crumbLabel: 'main' },
+    { pathname: '/blog', crumbLabel: 'blog' },
+    { pathname: '/blog/howto-blog-post/', crumbLabel: 'howto-blog-post' },
+  ],
+  customLabel: 'How to do a blog post',
+};
+
+export const WithHeader = Template.bind({});
+WithHeader.args = {
+  breadcrumbEntries: [
+    { pathname: '/', crumbLabel: 'main' },
+    { pathname: '/blog', crumbLabel: 'blog' },
+    { pathname: '/blog/howto-blog-post/', crumbLabel: 'howto-blog-post' },
+  ],
+  customLabel: 'How to do a blog post',
+};
+WithHeader.decorators = [
+  (Story) => (
+    <>
+      {CustomHeader()}
+      <Story />
+    </>
+  ),
+];
+
+export const WithHero = Template.bind({});
+WithHero.args = {
+  ...Regular.args,
+  hasHero: true,
+};
+WithHero.decorators = [
+  (Story) => (
+    <>
+      {CustomHeader(true)}
+      <img style={{ width: '100%' }} alt={''} src={AngularIMG} />
+      <Story />
+    </>
+  ),
+];
```

Comment:
All these stories are needed because the Breadcrumb knows too much. If we treat the Breadcrumb component as a simple "dumb" component that knows nothing about where it's being used then we can drastically reduce the complexity in these stories. I think we should aim for as-simple-as-possible components that we can then use to assemble a nice UI. As written in my other comment I strongly recommend to remove the header-height and hero knowledge/dependencies from this component.